index.html

```diff
@@ -117,11 +112,11 @@ <h3 class="title" id="project">Project</h3>
                 <p class="title">Project & Hobbies</p>
                 <div class="carousel-item has-background">
                     <div class="title">hangryFixed</div>
-                    <img class="hangryFixed" src="deployed-site.gif" alt="https://idelmundo.github.io/hangryfixed/" width="640" height="310/">
+                    <img class="carousel-image hangryFixed" src="deployed-site.gif" alt="https://idelmundo.github.io/hangryfixed/">
                 </div>
                 <div class="carousel-item has-background">
                     <div class="title">Cars</div>
-                    <img class="is-background " src="rx7.jpg " alt="" width="640" height="310" />
```

Comment:
Set image width and height in CSS. Makes it easier t o make your stuff responsive.